parlai/core/worlds.py

```diff
@@ -42,22 +42,29 @@
 """
 
 import copy
+import queue
 import random
 import time
 
+from abc import ABC, abstractmethod
+from enum import auto, Enum
 from functools import lru_cache
-from typing import List, Dict, Any, Union
+from typing import List, Dict, Any, Union, Tuple, Optional
 
 try:
-    from torch.multiprocessing import Process, Value, Condition, Semaphore
+    from torch.multiprocessing import Process, Value, Semaphore, Queue
+    import torch
+    import torch.multiprocessing as mp
 except ImportError:
```

Comment:
????

Reply:
i need the `multiprocessing` module to set the start method, i can remove the `import torch` however

Reply:
the confusion was w.r.t. the except importerror. why do we have two different multiprocessing modules here?

Reply:
Ahh, well we want to use torch multiprocessing if we can but will resort to regular multiprocessing if it's not available. @stephenroller has pointed out to me that Mturk uses these worlds so we don't want people to need pytorch just to run mturk stuff